esp/src/src/ESPUtil.ts

```diff
@@ -429,13 +429,15 @@ export class UndefinedMemory extends UndefinedMemoryBase {
     }
 }
 
-export function Grid(pagination?, selection?, overrides?, compoundColumns?) {
+export function Grid(pagination?, selection?, overrides?, compoundColumns?, gridName?) {
     var baseClass = [];
     var params = {};
+    var rows = this.LocalStorage.prototype.getItem(gridName, "Number");
```

Comment:
Change to: `Number(localStorage.getItem(gridName))` or `+localStorage.getItem(gridName)`

Reply:
Also when the user changes the "rows" I don't see where you set it in local storage?

Reply:
1. Will make change and test. 
2. The change of the rows is located in a different call https://github.com/hpcc-systems/HPCC-Platform/blob/17506aeeb78a4c42162dee0e4a3abee49966de7e/esp/src/src/ESPUtil.ts#L306 that was working OK. The reason for this change is that if the user changed it and reloaded the page it did not pull the user preference row number from local storage.